addon_service/common/osf.py

```diff
@@ -17,6 +19,11 @@
     "has_osf_permission_on_resource",
 )
 
+_OSF_HMAC_USER_HEADER = "X-Requesting-User-URI"
+_OSF_HMAC_RESOURCE_HEADER = "X-Requested-Resource-URI"
+_OSF_HMAC_PERMISSIONS_HEADER = "X-Requested-Resource-Permissions"
+_OSF_HMAC_PERMISSIONS_DELIMITER = ";"  # see https://github.com/CenterForOpenScience/osf.io/blob/00a17ecdb666c07245a520de3ccd0f743822573b/osf/external/gravy_valet/auth_helpers.py#L66
+
```

Comment:
Does it make sense to have these come from a common set of local env vars shared by OSF and GV? Or am I overreacting?

Reply:
not a bad idea, and i agree with the urge to genericize... but there's enough logic that has to be shared i'm not sure we gain anything but complexity by moving these to more env vars (at least until we have the code shared on both sides)